assets/js/blocks/cart-checkout-shared/payment-methods/test/payment-methods.js

```diff
@@ -102,9 +102,10 @@ describe( 'PaymentMethods', () => {
 		wpDataFunctions
 			.dispatch( CART_STORE_KEY )
 			.invalidateResolutionForStore();
-		wpDataFunctions
-			.dispatch( CART_STORE_KEY )
-			.receiveCart( defaultCartState.cartData );
+		wpDataFunctions.dispatch( CART_STORE_KEY ).receiveCart( {
+			...defaultCartState.cartData,
+			payment_methods: [ 'cod', 'credit-card' ],
+		} );
```

Comment:
`defaultCartState` isn't a `CartResponse` object. We can use the `previewCart` object, which includes these payment methods: `[ 'cod', 'bacs', 'cheque' ]`

```js
wpDataFunctions.dispatch( CART_STORE_KEY ).receiveCart( {
		...previewCart,
		payment_methods: [ 'cod', 'credit-card' ],
} );
```